tests/migration/register_system_in_textmode.pm

```diff
@@ -0,0 +1,42 @@
+# SLE15 migration tests
```

Comment:
textmode also refers to run ncurses and YaST so need to disambiguate this.
we can deal in some point with internal function names.
this could be a `SUSEConnect_register_system`

Reply:
Yes, I renamed the function name.